Source/WebCore/css/CSSCustomPropertyValue.cpp

```diff
@@ -89,6 +93,15 @@ String CSSCustomPropertyValue::customCSSText() const
             m_stringValue = value->tokenRange().serialize();
         }, [&](const SyntaxValue& syntaxValue) {
             m_stringValue = serializeSyntaxValue(syntaxValue);
+        }, [&](const SyntaxValueList& syntaxValueList) {
+            StringBuilder builder;
+            auto separator = separatorCSSText(syntaxValueList.separator);
+            for (auto& syntaxValue : syntaxValueList.values) {
+                if (&syntaxValue != &syntaxValueList.values.first())
```

Comment:
I personally think this would have looked a bit nicer:
`if (!builder.isEmpty())`

Pretty common pattern in WebKit code base too.